src/js/providers/html5.ts

```diff
@@ -809,6 +810,14 @@ function VideoProvider(this: HTML5Provider, _playerId: string, _playerConfig: Ge
         if (!_canSeek) {
             _canSeek = !!_getSeekableEnd();
         }
+
+        // Workaround for a bug in iOS where you are unable to seek in initially muted streams
+        if (OS.iOS && _videotag.muted && !iosMuteToggle) {
+            _videotag.muted = false;
+            _videotag.muted = true;
```

Comment:
>  It can also be set on canPlay, but I didn't think it should fire for every muted iOS video since it only pertains to seeking.

This will toggle mute only when `jwplayer().seek()` is called (using the API or via jwplayer's UI). This code doesn't run when or prior to seeking by setting `currentTime` directly on the media element.